### PR TITLE
SFI Remediation | Global Admin

### PR DESCRIPTION
**Pull request by seesharprun**

### CODE REVIEW

File: powerbi-ps/MicrosoftPowerBIMgmt.Workspaces/Restore-PowerBIWorkspace.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Restore-PowerBIWorkspace [-Scope <PowerBIUserScope>] [-RestoredName <String>] -A
 ## DESCRIPTION
 Restores a deleted Power BI workspace with the owner set to the user that matches the specified email address.
 Only workspaces in the new workspace experiences are supported and only deleted workspaces that haven't reached the data retention policy to be completely removed.
-Only -Scope Organization is supported and you must have administrator rights (such as Office 365 Global Administrator or Power BI Service Administrator) to call this command.
+Only -Scope Organization is supported and you must have the appropriate permissions or role (such as ower BI Service Administrator) to call this command.
```
Comment (SnehaGunda) [inline powerbi-ps/MicrosoftPowerBIMgmt.Workspaces/Restore-PowerBIWorkspace.md]:
```suggestion
Only -Scope Organization is supported and you must have the appropriate permissions or role (such as Power BI Service Administrator) to call this command.
```

File: powerbi-ps/MicrosoftPowerBIMgmt.Workspaces/Set-PowerBIWorkspace.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ Set-PowerBIWorkspace [-Scope <PowerBIUserScope>] -Id <Guid> -CapacityId <Guid> [
 ## DESCRIPTION
 Updates the name or description of a Power BI workspace.
 Only workspaces in the new workspace experiences preview are supported.
-Only -Scope Organization is supported and you must have administrator rights (such as Office 365 Global Administrator or Power BI Service Administrator) to call this command.
+Only -Scope Organization is supported and you must have the appropriate permissions or role (such as ower BI Service Administrator) to call this command.
```
Comment (SnehaGunda) [inline powerbi-ps/MicrosoftPowerBIMgmt.Workspaces/Set-PowerBIWorkspace.md]:
```suggestion
Only -Scope Organization is supported and you must have the appropriate permissions or role (such as Power BI Service Administrator) to call this command.
```

**Comment by seesharprun** [comment]
Thanks @SnehaGunda, I was moving too fast!

**Comment by maggiesMSFT** [comment]
Thanks, @seesharprun 
#sign-off

**Comment by seesharprun** [comment]
@maggiesMSFT, I don't think this repo is automated.

**Comment by seesharprun** [comment]
@MicrosoftDocs/learn-pr-review-team or @MicrosoftDocs/powerbi-docs-admin , can you help get this merged?

**Comment by rmca14** [comment]
@seesharprun The PR Review Team doesn't handle this repo, nor do we have merge perms.

**Comment by seesharprun** [comment]
> @seesharprun The PR Review Team doesn't handle this repo, nor do we have merge perms.

Thanks for the clarification. @MicrosoftDocs/powerbi-docs-admin & @MicrosoftDocs/powerbi-docs-write, can any of you merge?

**Comment by seesharprun** [comment]
@huypub

**Pull request merged**